Todo.WillowBear/Todo.WillowBear/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Serilog;
+using Todo.WillowBear.Data;
+;
+
+
+//add serilog
+Log.Logger = new LoggerConfiguration()
+    .MinimumLevel.Debug()
+    .WriteTo.Console()
+    .CreateLogger();
+
+try
+{
+    Log.Information("Starting web application");
+
+    var builder = WebApplication.CreateBuilder(args);
+
+    // Add services to the container.
```

Comment:
Again, comments are ok for learning, but not for pushing to production

Reply:
Fixed.